ArkoudaDemoBasics.ipynb

```diff
@@ -0,0 +1,425 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Arkouda\n",
+    "## Comparison to Numpy/Pandas\n",
+    "\n",
+    "The majority of the API for arkouda functions very similarly to that of Numpy or Pandas. However, under the hood, Arkouda is sending messages out to the Chapel Server for processing. This demo will highlight the key similarities between Arkouda and NumPy/Pandas. Additionally, we will explore some of the chapel functionality making this possible."
```

Comment:
I don't think this notebook gets into the chapel functionality. I believe the original demo for this notebook had an exploration of that